commands.js

```diff
@@ -5,8 +5,201 @@
  * These are system commands - commands required for Pokemon Showdown
  * to run. A lot of these are sent by the client.
  *
- * If you'd like to modify commands, please go to config/commands.js,
- * which also teaches you how to use commands.
+ * If you'd like to modify commands, please go to the chat-plugins
+ * folder.
+ *
+ * You can define the commands such as 'whois', then use it by
+ * typing /whois into Pokemon Showdown.
+ *
+ * A command can be in the form:
+ *   ip: 'whois',
+ * This is called an alias: it makes it so /ip does the same thing as
+ * /whois.
+ *
+ * But to actually define a command, it's a function:
+ *
+ *   allowchallenges: function (target, room, user) {
+ *     user.blockChallenges = false;
+ *     this.sendReply("You are available for challenges from now on.");
+ *   }
```

Comment:
Sure. Should it still be a relatively short one even if not much happens in it?
How about using `/avatars` or `/calc` ?